.scripts/publish_storybook.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
Could we use something like [gh-pages](https://github.com/tschaub/gh-pages) for this?

Reply:
This looks good, I'm going to create a separate issue for addressing this in the future.

Reply:
https://github.com/resin-io-modules/resin-components/issues/39